docs/pages/feedback/feedback-page.md

```diff
@@ -185,9 +191,13 @@ const FeedbackPage = ({ initialFeedback }) => {
           </FormControl>
         </Box>
       )}
+
       {allFeedback &&
         allFeedback.map((feedback) => (
-          <Feedback key={feedback.id} {...feedback} />
+          <Feedback
+            key={feedback.id || new Date().getTime().toString()}
```

Comment:
Why was this added?

Reply:
React complains about the `undefined` key when the feedback was created using the UI (as opposed to loading from Firebase). I wasn't sure this was the best approach given we want to display the new feedback before firebase persists it.

Reply:
At a certain point in the course, we change this so that it gets the ID from Firebase of the new document async. I might have just not added it to the docs. Hmm.